__tests__/workers/personalizedDigestEmail.ts

```diff
@@ -0,0 +1,169 @@
+import { expectSuccessfulBackground, saveFixtures } from '../helpers';
```

Comment:
I know, but I would have to hardcode bunch of values and computations in there which would again be sensitive with all the data changes. So snapshot looked like the easiest solution.

Reply:
Because I wanted to make sure all the properties are there and correct when sending an email, if some of them are wrong we can potentially have thousands of invalid emails.

Reply:
If we wanna avoid snapshots I would just copy their current values in raw in code and use `toMatchObject`

Reply:
Let's leave it as it is for now then